.remarkrc.js

```diff
@@ -1,5 +1,14 @@
 // Configuration file used to find popular mistakes in .md files.
 
+import remarkLintTitleCapitalization from './remark-custom-rules/title-capitalization/remarkLintTitleCapitalization.js'
+
 export default {
-  plugins: ['remark-lint-no-dead-urls', 'remark-validate-links']
+  plugins: [
+    'remark-lint-no-dead-urls',
+    'remark-validate-links',
+    [
+      remarkLintTitleCapitalization,
+      { exceptionWords: { '2FA': true, 'GitHub': true } }
```

Comment:
The wrong upper case letter inside the word is very rare use case.

What if will not check it? Only the first letter of the word?